FirebaseInAppMessaging/Swift/Source/SwiftUIPreviewHelpers.swift

```diff
@@ -14,6 +14,8 @@
 
 import FirebaseInAppMessaging
 
+@available(iOS 13, tvOS 13, *)
+@available(iOSApplicationExtension, unavailable)
```

Comment:
I'm going to have troubles cherry-picking this since this was added post code freeze as far as I recall

Reply:
I can merge a cherry-pick if we decide to do so.